io/plugins/eu.esdihumboldt.hale.io.jdbc.ui/src/eu/esdihumboldt/hale/io/jdbc/ui/SchemasRetrievalPage.java

```diff
@@ -135,7 +133,7 @@ protected void onShowPage(boolean firstShow) {
 				disposeControl();
 				// message
 				Label label = new Label(page, SWT.WRAP);
-				label.setText("Schema retrieval is not enabled for selected driver.");
+				label.setText("Schema selection is not enabled for selected driver.");
```

Comment:
I still think it would be better to give more information to the user. What does this actually mean to the user? He might get the idea that he should do something to enable it.

I revise my previous suggestion to this:
"Restricting the tables to load to tables from specific database schemas is not supported for this database driver. All database tables will be loaded."